tendrl/gluster_integration/objects/volume/atoms/check_volume_available/__init__.py

```diff
@@ -5,6 +5,7 @@
 from tendrl.commons import objects
 from tendrl.commons.objects import AtomExecutionFailedError
 from tendrl.commons.utils import log_utils as logger
+from tendrl.gluster_integration.objects.volume import Volume
```

Comment:
Please use NS.gluster.objects.Volume instead of import

Reply:
Done. Will remove all the atoms which are not used at the moment.